### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by rkterungwa16**
Please guys, review my test cases

### CODE REVIEW

**Comment by tduyile04** [comment]
- Ensure search can handle varying number of search terms ( handling an array of search terms )..
- Ensure that the passed in index is in the correct form
Good test-scope coverage, nice job man

**Comment by OnyekachiSamuel** [comment]
Good job but your implementation of your search on string combination is meant to return search result
for the individual words. Also the use of fileName with global scope is confusing because you accessed it as an array. Good job.

File: tests/inverted-index-testSpec.js [hunk 1]
```diff
@@ -1,7 +1,201 @@
...
+const validJson = new myApp('bookone.json');
+const multipleFiles = new myApp(['book.json', 'bookone.json']);
+
+describe('Inverted index class', () => {
+  describe('Read book data', () => {
+    it("Should return 'malformed json' for fileContent = ''", () => {
+      expect(invalidJson.getJson()).toEqual('malformed json');
+    });
+
+    it("Should return '[{}]' for fileContent = '[{}]'", () => {
```
Comment (uenyingwa) [inline tests/inverted-index-testSpec.js]: It should return a proper error response and not just an empty object literal.

File: tests/inverted-index-testSpec.js [hunk 2]
```diff
@@ -1,7 +1,201 @@
...
+      expect(jsonOfFile.searchIndex('world..')).toEqual({ 'world': [1] });
+    });
+
+    it('Should return {"world": [1]} for searchIndex("WoRld")', () => {
+      expect(jsonOfFile.searchIndex('WoRld')).toEqual({ 'world': [1] });
+    })
+
+    it('Should return {"what": [1]} for searchIndex("what")', () => {
+      expect(validJson.searchIndex('what')).toEqual({ 'what': [ 0 ] });
+    });
```
Comment (uenyingwa) [inline tests/inverted-index-testSpec.js]: There should be a test case to ensure searchIndex goes through all indexed files if a filename/key is not passed.

**Review by uenyingwa — CHANGES_REQUESTED** [review]
Make changes and get back to me.

**Pull request closed without merging**